### ISSUE TITLE
Remove business logic and strings from Waitlist component

### ISSUE THREAD

**Issue by emilyjablonski** [bloom-housing/bloom#2688]
We should have no business logic or pre-translated strings in the Waitlist component, and allow styled description content.

Additionally the header doesn't meet color contrast requirements, so it needs to be made a little darker.

Here is the SF version for context:
<img width="355" alt="Screen Shot 2022-04-22 at 9 52 27 PM" src="https://user-images.githubusercontent.com/65367387/164874197-ec7b54f5-6568-4535-960d-0e4122f8102b.png">

**Comment by emilyjablonski** [comment]
QA Notes: TBD, waiting on design review

**Comment by slowbot** [comment]
@emilyjablonski LGTM
